modules/ROOT/pages/_partials/proc_self-signed-certs.adoc

```diff
@@ -29,26 +28,25 @@ base64 --decode | sed -e '1,/^-----END RSA PRIVATE KEY-----$/ d'  > /tmp/localcl
 The resulting file is located in the `/tmp` directory.
 
 [[installing-on-device]]
-== Installing the OpenShift Root Certificate Authority Cert on a Device
+== 2. Installing the OpenShift Root Certificate Authority Cert on a Device
+
+[role="primary"]
+.Android
 
-. Configure a pin to ensure the certificate can be installed.
+****
+.. Enable PIN protection on the mobile device to ensure the certificate can be installed.
 
-. Copy the file to your device using one of the following methods:
+.. Copy the file to your device using one of the following methods:
 +
-* USB *(Android only)* - Attach the device to the machine hosting
+* USB - Attach the device to the machine hosting
```

Comment:
Do we want to a link to android documents for how to copy a file to Android using USB? The current instructions tell you what to do, but not how to do them.

Reply:
We can, but it could be different depending on the manufacture. So here I will link to the guide for Google Nexus and use it as an example.